bridge/client/app/_models/trace.ts

```diff
@@ -132,10 +121,10 @@ class Trace {
   getLabel(): string {
     // TODO: use translation file
```

Comment:
Is this TODO still relevant?

Reply:
Yes, there is even an issue for this #1339